SimCalorimetry/HGCalSimProducers/interface/HGCDigitizer.h

```diff
@@ -11,7 +11,7 @@
 
 #include "SimCalorimetry/HGCalSimProducers/interface/HGCDigitizerBase.h"
 #include "DataFormats/HGCDigi/interface/HGCDigiCollections.h"
-#include "DataFormats/HGCDigi/interface/PHGCSimAccumulator.h"
+#include "DataFormats/HGCalDigi/interface/PHGCSimAccumulator.h"
```

Comment:
Could you please clarify what is the different of HGCDigi and HGCalDigi subpackages?

Reply:
I would like to dephase HGCDigi in the future and use a name which is more inline with the one used the SimProducers/HGCal{SimAlgos,SimProducers} i.e. starting with HGCal not HGC

The PHGCSimAccumulator.h class will still be used so I moved it's location, but the new DataFormat structure (HGCROCChannelDataFrame.h) is totally different